### ISSUE TITLE
Expose a way to get the roomId from the result of a "query.room"

### ISSUE THREAD

**Issue by Half-Shot** [turt2live/matrix-js-bot-sdk#51]
In matrix-appservice-bridge, there is a callback for "onAliasQueried" which returns the alias and roomId. This is useful for sending normal events into the room once it's been created.

**Comment by turt2live** [comment]
Listen for the join event on the room and check the room alias?

**Comment by Half-Shot** [comment]
Yup, but an emitter would be cleaner and it's pretty much a one liner to do.

**Comment by turt2live** [comment]
Definitions of cleaner are subjective :p 

The real reason this hasn't been done is because I'm not sure if I want to encourage the workflow from the existing bridge-sdk in this sdk.
